lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -30,7 +30,7 @@
                         breakpoints: this.breakpoints
                     });
 
-                    if ( this.model.get('name') ) {
+                    if ( this.model.get('programData').name ) {
```

Comment:
@AlasdairSwan yup, this fails with `TypeError: Cannot read property 'name' of undefined` if `programData` is missing from the model. What's the intent of this conditional? Would you suggest I change it to check if `programData` is defined?
```js
if ( this.model.get('programData') ) {
    ...
```